index.d.ts

```diff
@@ -42,9 +47,15 @@ hexRgb('#cd2222cc');
 
 hexRgb('#cd2222cc', {format: 'array'});
 //=> [205, 34, 34, 0.8]
+
+hexRgb('#cd2222cc', {format: 'css'});
+//=> 'rgb(205 34 34 / 80%)'
 ```
 */
 declare function hexRgb(hex: string): hexRgb.RgbaObject;
+
+declare function hexRgb(hex: string, options: hexRgb.Options): hexRgb.RgbaObject;
 declare function hexRgb(hex: string, options: hexRgb.Options): hexRgb.RgbaTuple;
+declare function hexRgb(hex: string, options: hexRgb.Options): string;
```

Comment:
This is not correct. Like I said, the return value should be overloaded based on the format option value.

Reply:
I hope f7ee876 is what you are looking for, if not I'm afraid I don't know how to achieve that 🤔

Reply:
Yes, that's what I had in mind.